vanilla-framework/vanilla-framework
# Update docs/contextual-menu page content

## Done
- Updated copy
- Updated demo example to showcase live example 😉 
- Check content matches copy doc [here](https://docs.google.com/document/d/1FK485n4m21iCU_yt3DrsAcBPeiukpWUUQZLX4_PGXSM/edit#heading=h.m8r6djutwkm5)

## QA
- Pull code
- Run /docs > ./run 
- Check http://0.0.0.0:8104/en/patterns/contextual-menu/
- Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/contextual-menu/
- Review updated example

## Details
- Fixes https://github.com/ubuntudesign/vanilla-design/issues/344
